src/cmd/clean.go

```diff
@@ -25,12 +29,26 @@ var cleanCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		fmt.Println("removed all dangling images")
+		logger.Log("exo-clean", "removed all dangling images")
```

Comment:
This could be `logChannel <- "removed all dangling images"`

Reply:
I think we avoided using logChannels here because that gave us a problem where the process would finish before all statements were printed.

Reply:
True, nevermind good as is